alterations/alterations_test.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+    "testing"
+    // "fmt"
+    "github.com/OWASP/Amass/v3/alterations"
+)
+
+var subdomain = "fa2ke.test.com"
+
+func check_alterations(generated_subdomains []string, expected_subdomains []string) int {
+    total_results := 0
+    for _, label := range generated_subdomains {
+        for _, results := range expected_subdomains {
+            if results == label {
+                total_results++
+            }
+        }
+    }
+    return total_results

```

Comment:
You could just calculate the `len(expected_subdomains)` here and return a `bool` to show whether the two slices are the same.